styles/base.css

```diff
@@ -273,3 +320,126 @@ h6 {
         width: calc(181.676rem / 16);
     }
 }
+
+/* Utility Classes  */
+
+.fs-7 { /* Font size smaller than 1rem */
+    font-size: calc(14rem / 16) !important;
+}
+
+.bg-w-100 { /* Full-width background color, goes beyond gutter */
+    margin: 0 calc(var(--bs-gutter-x) * -.5);
+    padding: 0 calc(var(--bs-gutter-x) * .5);
+}
+
+.border-100 { /* Full-width border, goes beyond gutter */
+    margin: 0 calc(var(--bs-gutter-x) * -.5);
+    padding: 0;
+}
+
+.border-half {
+    --bs-border-width: 0.5px;
+}
+
+.border-black {
+    --bs-border-color: #000;
+}
+
+.list-circle {
+    background: url(/assets/illustrations/circle.svg) no-repeat left center;
+    padding: 0.5rem 0 0.5rem 2.4rem;
+    list-style: none;
+    margin: 0;
+}
+
+/* About */
+
+@-webkit-keyframes move-left
+{
+    27.27%, 45.45% {
+        right: 60%;
+    }
+    63.63%, 100% {
+        right: calc(100vw + 62px);
+    }
+}
+@keyframes move-left
+{
+    27.27%, 45.45% {
+        right: 60%;
+    }
+    63.63%, 100% {
+        right: calc(100vw + 62px);
+    }
+}
+
+.animate-bus {
+    right: -62px;
+    bottom: 0;
+    position: absolute;
+    -webkit-animation: move-left 22s linear infinite;
+    animation: move-left 22s linear infinite;
+}
```

Comment:
The animation requirements I got from Segacy were: Move for 6 seconds, pause for 4 seconds, move for 4 seconds for the total animation, then have an 8 second pause in between loops. Each total cycle is 22 seconds, which includes the pause. Lastly, she asked that the bus travel at the same speed during both its 2 segments.

This was the math I did to arrive at these percentages. The first numbers (`27%, 45%`) are an expression of _times_, as percentages of the total 22 second loop. The last number (`60%`), is how far the object travels along the x-axis, so 60% means 60% of the viewport width. Because the overall _speed_ of the two "move" rows are the same, at 10 (percent of viewport width per second), the bus should look like it's going the same speed throughout the page.

So the bus takes 6 seconds (from 0 to 27%) to go 60% of the width, then pauses there for 4 seconds (which is expressed as a total 45% elapsed time). Then, it starts again from the 60% width point to the end in 4 seconds. Lastly, it pauses _off_ the page, expressed as `calc(100vw + width of the bus);` for 8 seconds. 

duration |   | time in seconds | time in percent |   | location start | location end | total distance | speed
-- | -- | -- | -- | -- | -- | -- | -- | --
  |   | sum | (time / total) % |   | x-axis % | x-axis % | (start-end) | (distance/seconds)
6 | move | 6 | 27.27272727 |   | 0 | 60 | 60 | 10
4 | pause | 10 | 45.45454545 |   | 60 | 60 | 0 | 0
4 | move | 14 | 63.63636364 |   | 60 | 100 | 40 | 10
8 | pause | 22 | 100 |   | 100 | 100 | 0 | 0

I hope I did the math right 😅

Reply:
This is intense 🤯